prodtests/full-system-test/run-workflow-on-inputlist.sh

```diff
@@ -68,7 +73,8 @@ start_process $MYDIR/dpl-workflow.sh
 
 if [[ "0$4" != "00" ]]; then
   sleep 1
-  tail -f ${LOG_PREFIX}*.log &
+  tail -n 1000000 -f ${LOG_PREFIX}*.log &
+  ln -sf ${LOG_PREFIX}*.log latest.log
```

Comment:
The `ln` with `*` will fail in DD mode, since in that case there are 2 log files

Reply:
I see. I would like to link to a file that always points to the latest log.

Reply:
Could you try `ln -sf ${LOG_PREFIX}${NUM_PROCS}.log latest.log` ?

Reply:
Done and working